app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,99 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.dto.Task;
+import com.codesoom.assignment.generators.IdGenerator;
+import com.codesoom.assignment.repositories.TaskRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+/**
+ * 할 일에 대한 HTTP request를 처리하는 컨트롤러다.
+ */
+@RestController
+@RequestMapping(path = "/tasks")
+@CrossOrigin
+public class TaskController {
+
+    private final TaskRepository taskRepository;
+    private final IdGenerator taskIdGenerator;
+
+    @Autowired
+    public TaskController(TaskRepository taskRepository) {
+        this.taskRepository = taskRepository;
+        this.taskIdGenerator = new IdGenerator();
+    }
+
+    /**
+     * 모든 할 일의 리스트를 반환한다.
+     *
+     * @return 할 일 목록 전체를 반환한다.
+     */
```

Comment:
`@return`에서는 리턴되는 값의 타입에 좀 더 집중해 봅시다. 그리고 `@param` 이나 `@return`은 summary보다 더 짧고 단순한 표현을 사용해 봅시다.

```suggestion
    /**
     * 모든 할 일의 리스트를 반환한다.
     *
     * @return 할 일 목록
     */
```

Reply:
`@return` 은 짧고 단순한 표현을 사용해 반환되는 것이 '무엇'인지를 명확히 하도록 하겠습니다.